docs/build/unitary-synthesis.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Synthesizing unitary matrices
+description: On the implementation of arbirtary unitary matrices on qubits
+---
+
```

Comment:
I think this is missing a title? It need a `# something`, otherwise it renders like this:
<img width="1382" alt="Screenshot 2023-11-28 at 15 27 10" src="https://github.com/Qiskit/documentation/assets/23662430/bfa5fe87-6514-4b03-98cb-457ce1b27d4b">

Reply:
Ah crap I thought the title in the `---` would be the header -- I'll add some